jbi/models.py

```diff
@@ -22,6 +22,13 @@
 JIRA_HOSTNAMES = ("jira", "atlassian")
 
 
+@functools.lru_cache
+def steps_module_has_function(function_name):
+    from jbi import steps
```

Comment:
I don't love that I had to do this late import (it's usually a code smell to me), but without this, I was running into circular imports.

Reply:
I think this could be acceptable in this particular case since we are introspecting modules, but yeah I see what you mean